include/deal.II/fe/mapping.h

```diff
@@ -365,6 +365,26 @@ class Mapping : public Subscriptor
   get_center(const typename Triangulation<dim, spacedim>::cell_iterator &cell,
              const bool map_center_of_reference_cell = true) const;
 
+  /**
+   * Return the bounding box of a mapped cell.
+   *
+   * If you are using a (bi-,tri-)linear mapping that preserves vertex
+   * locations, this function simply returns the value also produced by
+   * `cell->bounding_box()`. However, there are also mappings that add
+   * displacements or choose completely different locations, e.g.,
+   * MappingQEulerian, MappingQ1Eulerian, or MappingFEField.
+   *
+   * This function returns the bounding box containing all the vertices of the
+   * cell, as returned by the get_vertices() method.
```

Comment:
Can you mention that in case of curved boundaries, this binding box does not necessarily include the entirety of the cell?